core/src/products/models.py

```diff
@@ -1 +1,45 @@
-# Create your models here.
+from django.db import models
+from categories.models import Category
+
+
+class Product(models.Model):
+    title = models.CharField(max_length=200)
```

Comment:
Its good to have uuid for each product, we can use it in URLs and APIs, meanwhile we can use id in internal DB relations
```suggestion
    id = models.AutoField(primary_key=True)
    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
    title = models.CharField(max_length=200)
```